### PR TITLE
[hi] Localized content/en/docs/tasks/administer-cluster/access-cluster-api.md in hindi 

### PR DESCRIPTION
**Pull request by KashishLakhara04**
fixes: #40757

### CODE REVIEW

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
+---
+title: Kubernetes API का उपयोग करके क्लस्टर एक्सेस करें
```
Comment (Babapool) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:2]:
```suggestion
title: कुबेरनेट्स API का उपयोग करके क्लस्टर एक्सेस करें
```

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+content_type: task
+weight: 60
+---
+
+<!-- overview -->
+यह पेज Kubernetes API का उपयोग करके क्लस्टर एक्सेस करने का तरीका दिखाता है।
```
Comment (Babapool) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:8]:
```suggestion
यह पृष्ठ कुबेरनेट्स API का उपयोग करके क्लस्टर एक्सेस करने का तरीका दिखाता है।
```

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Kubernetes API एक्सेस करना
```
Comment (Babapool) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:16]:
```suggestion
## कुबेरनेट्स API एक्सेस करना
```

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+
+### kubectl से पहली बार एक्सेस करना
+
+Kubernetes API को पहली बार एक्सेस करते समय, Kubernetes कमांड-लाइन टूल `kubectl` का उपयोग करें।
```
Comment (Babapool) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:20]:
```suggestion
कुबेरनेट्स API को पहली बार एक्सेस करते समय, कुबेरनेट्स कमांड-लाइन टूल `kubectl` का उपयोग करें।
```

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+
+किसी क्लस्टर को एक्सेस करने के लिए आपको क्लस्टर का लोकेशन जानना होगा और क्रेडेंशियल होना चाहिए
+
+इसे एक्सेस करने के लिए।आमतौर पर, जब आप [गेटिंग स्टार्ट गाइड](/hi/docs/setup/) के माध्यम से काम करते हैं तो यह स्वचालित रूप से सेट-अप हो जाता है या किसी और ने क्लस्टर स्थापित किया और आपको क्रेडेंशियल्स और एक लोकेशन प्रदान किया।
```
Comment (Babapool) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:24]:
```suggestion
आमतौर पर, यह स्वचालित रूप से सेट-अप हो जाता है जब आप [गेटिंग स्टार्ट गाइड](/hi/docs/setup/) के माध्यम से काम करते हैं तो या किसी और ने क्लस्टर स्थापित किया और आपको क्रेडेंशियल्स और एक लोकेशन प्रदान किया।
```

**Review by Babapool — COMMENTED** [review]
Some suggestions from myside, hope you find them useful.  Thanks

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+
+कुछ क्लस्टर्स पर, API सर्वर को प्रमाणीकरण की आवश्यकता नहीं होती है; यह लोकलहोस्ट पर काम कर सकता है, या फ़ायरवॉल द्वारा सुरक्षित हो सकता है। इसके लिए कोई मानक नहीं है। [Kubernetes API तक पहुंच को नियंत्रित करना](/docs/concepts/security/controlling-access)
+वर्णन करता है कि आप इसे क्लस्टर एडमिनिस्ट्रेटर के रूप में कैसे कॉन्फ़िगर कर सकते हैं।
+
+### API के लिए प्रोग्रामेटिक एक्सेस
+
+कुबेरनेट्स आधिकारिक तौर [Go](#go-क-ल-इ-ट), [Python](#python-क-ल-इ-ट), [Java](#java-क-ल-इ-ट), [dotnet](#dotnet-क-ल-इ-ट), [JavaScript](#javascript-क-ल-इ-ट), and [Haskell](#haskell-क-ल-इ-ट) के लिए क्लाइंट लाइब्रेरी का समर्थन करता है। अन्य क्लाइंट लाइब्रेरी हैं जो उनके लेखकों द्वारा प्रदान की जाती हैं और बनाए रखी जाती हैं, न कि कुबेरनेट्स टीम। अन्य लैंग्वेजेज से API एक्सेस और वे कैसे प्रमाणित करते हैं, इसके लिए [क्लाइंट लाइब्रेरी](/docs/reference/using-api/client-libraries/) देखें।
```
Comment (Babapool) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:135]:
```suggestion
कुबेरनेट्स आधिकारिक तौर पर [Go](#go-client), [Python](#python-client), [Java](#java-client), [dotnet](#dotnet-client), [JavaScript](#javascript-client) और [Haskell](#haskell-client) के लिए क्लाइंट लाइब्रेरी का समर्थन करता है। कुछ अन्य क्लाइंट लाइब्रेरी भी हैं जो कुबेरनेट्स टीम के बजाय संबंधित लेखकों द्वारा प्रदान और रखरखाव की जाती हैं। अन्य लैंग्वेजेज से API एक्सेस और वे कैसे प्रमाणित करते हैं, इसके लिए [क्लाइंट लाइब्रेरी](/docs/reference/using-api/client-libraries/) देखें।
```

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 7]
```diff
@@ -0,0 +1,320 @@
...
+
+#### Go क्लाइंट
+
+* लाइब्रेरी प्राप्त करने के लिए, निम्न कमांड रन करें: `go get k8s.io/client-go@kubernetes-<kubernetes-version-number>` कौन से संस्करण समर्थित हैं, यह देखने के लिए [https://github.com/kubernetes/client-go/releases](https://github.com/kubernetes/client-go/releases) देखें।
```
Comment (Babapool) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:139]:
```suggestion
* लाइब्रेरी प्राप्त करने के लिए, निम्न कमांड रन करें: `go get k8s.io/client-go@kubernetes-<kubernetes-version-number>` यह देखने के लिए कि कौन से संस्करण समर्थित हैं  [https://github.com/kubernetes/client-go/releases](https://github.com/kubernetes/client-go/releases) देखें।
```

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 8]
```diff
@@ -0,0 +1,320 @@
...
+* क्लाइंट-Go क्लाइंट के ऊपर एक एप्लिकेशन लिखें।
```
Comment (Babapool) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:140]:
```suggestion
* client-go क्लाइंट के ऊपर एक एप्लिकेशन लिखें।
```

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 9]
```diff
@@ -0,0 +1,320 @@
...
+
+{{< note >}}
+
+क्लाइंट-Go अपने स्वयं के API ऑब्जेक्ट्स को परिभाषित करता है, इसलिए यदि आवश्यक हो, तो मुख्य रिपॉजिटरी के बजाय क्लाइंट-गो से एपीआई परिभाषाएं आयात करें। उदाहरण के लिए, `import "k8s.io/client-go/kubernetes"` सही है।
```
Comment (Babapool) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:144]:
```suggestion
client-go स्वयं से API ऑब्जेक्ट्स को परिभाषित करता है, इसलिए यदि आवश्यक हो, तो मुख्य रिपॉजिटरी के बजाय client-go से API परिभाषाएं आयात करें। उदाहरण के लिए, `import "k8s.io/client-go/kubernetes"` सही है।
```

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 10]
```diff
@@ -0,0 +1,320 @@
...
+
+{{< /note >}}
+
+Go क्लाइंट उसी [kubeconfig फ़ाइल](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) का उपयोग कर सकता है
+जैसा कि kubectl CLI API सर्वर का पता लगाने और प्रमाणित करने के लिए करता है। यह [उदाहरण](https://git.k8s.io/client-go/examples/out-of-cluster-client-configuration/main.go) देखें:
```
Comment (Babapool) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:149]:
```suggestion
जिसे  कि kubectl CLI API सर्वर का पता लगाने और प्रमाणित करने के लिए करता है। यह [उदाहरण](https://git.k8s.io/client-go/examples/out-of-cluster-client-configuration/main.go) देखें:
```

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 11]
```diff
@@ -0,0 +1,320 @@
...
+  config, _ := clientcmd.BuildConfigFromFlags("", "<path-to-kubeconfig>")
+  //क्लाइंटसेट बनाता है
+  clientset, _ := kubernetes.NewForConfig(config)
+  // पॉड्स को सूचीबद्ध करने के लिए API एक्सेस करें
+  pods, _ := clientset.CoreV1().Pods("").List(context.TODO(), v1.ListOptions{})
+  fmt.Printf("There are %d pods in the cluster\n", len(pods.Items))
+}
+```
+
+यदि एप्लिकेशन को क्लस्टर में पॉड के रूप में तैनात किया गया है, तो [पॉड के भीतर से एपीआई एक्सेस करना](/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod) देखें।
```
Comment (Babapool) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:173]:
```suggestion
यदि एप्लिकेशन को क्लस्टर में पॉड के रूप में डिप्लॉय किया गया है, तो [पॉड के भीतर से API एक्सेस करना](/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod) देखें।
```

**Review by Babapool — COMMENTED** [review]
Some suggestions from myside, hope you find them useful. Thanks

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 12]
```diff
@@ -0,0 +1,320 @@
...
+#### kubectl प्रॉक्सी के बिना
+
+API सर्वर को सीधे एक प्रमाणीकरण टोकन पास करके कुबेक्टल प्रॉक्सी का उपयोग करने से बचना संभव है, जैसे:
+
+`grep/cut` दृष्टिकोण का उपयोग करना:
+
+```shell
+# सभी संभावित क्लस्टर्स की जाँच करें, क्योंकि आपके .KUBECONFIG में कई संदर्भ हो सकते हैं:
+kubectl config view -o jsonpath='{"Cluster name\tServer\n"}{range .clusters[*]}{.name}{"\t"}{.cluster.server}{"\n"}{end}'
+# परोक्त आउटपुट से आप जिस क्लस्टर से इंटरैक्ट करना चाहते हैं उसका नाम चुनें:
```
Comment (kundan2707) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:87]: उपरोक्त आउटपुट से आप जिस क्लस्टर से इंटरैक्ट करना चाहते हैं उसका नाम चुनें

File: content/hi/docs/tasks/administer-cluster/access-cluster-api.md [hunk 13]
```diff
@@ -0,0 +1,320 @@
...
+    {
+      "clientCIDR": "0.0.0.0/0",
+      "serverAddress": "172.30.1.2:6443"
+    }
+  ]
+}
+```
+उपरोक्त उदाहरण `--insecure` फ्लैग का उपयोग करता है। यह इसे MITM अटैक के अधीन छोड़ देता है। जब kubectl क्लस्टर एक्सेस करता है तो यह सर्वर एक्सेस करने के लिए संग्रहीत रूट प्रमाणपत्र और क्लाइंट प्रमाणपत्र का उपयोग करता है। (ये `~/.kube` डायरेक्टरी में संस्थापित हैं) चूँकि क्लस्टर प्रमाणपत्र आमतौर पर स्व-हस्ताक्षरित होते हैं, इसलिए आपके http क्लाइंट को रूट प्रमाणपत्र का उपयोग करने के लिए विशेष कॉन्फ़िगरेशन की आवश्यकता हो सकती है।
+
+कुछ क्लस्टर्स पर, API सर्वर को प्रमाणीकरण की आवश्यकता नहीं होती है; यह लोकलहोस्ट पर काम कर सकता है, या फ़ायरवॉल द्वारा सुरक्षित हो सकता है। इसके लिए कोई मानक नहीं है। [Kubernetes API तक पहुंच को नियंत्रित करना](/docs/concepts/security/controlling-access)
```
Comment (kundan2707) [inline content/hi/docs/tasks/administer-cluster/access-cluster-api.md:130]:
```suggestion
कुछ क्लस्टर्स पर, API सर्वर को प्रमाणीकरण की आवश्यकता नहीं होती है। यह लोकलहोस्ट पर काम कर सकता है, या फ़ायरवॉल द्वारा सुरक्षित हो सकता है। इसके लिए कोई मानक नहीं है। [Kubernetes API तक पहुंच को नियंत्रित करना](/docs/concepts/security/controlling-access)
```

**Comment by aj11anuj** [comment]
Since, a long time there isn't any respond/activity from your side on this PR @KashishLakhara04. Kindly, respond or implement changes so that this PR can either be merged or closed.

**Comment by divya-mohan0209** [comment]
@KashishLakhara04 : I'll be closing off this issue since there has been no response for ~8 months. 
/close

**Pull request closed without merging**